projects/packages/forms/src/dashboard/components/table/index.js

```diff
@@ -13,7 +13,7 @@ const Table = ( { className, columns, defaultSelected, items, onSelectionChange
 			const newState = includes( selected, id ) ? without( selected, id ) : [ ...selected, id ];
 
 			setSelected( newState );
-			onSelectionChange( newState );
+			onSelectionChange( last( newState ) );
```

Comment:
Why are we changing this? The idea behind it was to return the `ID`s of all selected elements, but now we're returning only the last one that was selected?  
Also the 'unselecting' behavior seems confusing with this change considering this is a toggling function.

Reply:
Since this is resolved, I assume that's something you've discussed in Slack?